### ISSUE TITLE
Falcosidekick outout KAFKA with TLS support

### ISSUE THREAD

**Issue by victorrodriguez1984** [falcosecurity/falcosidekick#547]
**Motivation**

**Feature**

**Alternatives**

**Comment by Issif** [comment]
Have you tested this config?:
```yaml
kafka:
  hostport: "ssl://<address>:9092"
```

Technically, it should work.

**Comment by Issif** [comment]
@victorrodriguez1984 can you try my proposal to check if it's ok please?

**Comment by ibice** [comment]
Hi @Issif, @victorrodriguez1984 and me have tested your proposal, but it seems it doesn't like it. It tries to resolve `ssl://<address>:<port>`. Here's the error message:

```console
dial tcp: lookup ssl://server:9092: no such host
```

To see the error, I had to modify the code to handle this returned err:
https://github.com/falcosecurity/falcosidekick/blob/4d11678483cf3cd4703f50f54a69c76a778163ac/outputs/kafka.go#L142-L143
It might be worth handling it, it seems `handleKafkaCompletion` captures errors during async writes and `WriteMessages` returns synchronous errors. What do you think?

**Comment by Issif** [comment]
Everything is available with release 2.28.0 :+1:
